bindings/kotlin/example/app/src/main/kotlin/com/lni/example/MainActivity.kt

```
@@ -0,0 +1,553 @@
+
+        sb.appendLine("Fetching balance...")
+
+        val info = node.getInfo()
+        
+        // Convert from millisats to sats for display
+        val balanceSats = info.sendBalanceMsat / 1000
+        val balanceBtc = info.sendBalanceMsat / 100_000_000_000.0
+        
+        sb.appendLine("✓ Connected to Strike!\n")
+        sb.appendLine("Balance: $balanceSats sats")
+        sb.appendLine("         ${String.format("%.8f", balanceBtc)} BTC")
```

detekt (1.23.8)

[warning] String.format("%.8f", balanceBtc) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)